### PR TITLE
Remove duplicate `.gitignore` for `.DS_Store`

### PR DESCRIPTION
**Pull request by fumiya-kume**
## Issue
- N/A

## Overview (Required)
- Remove duplicate `.gitignore` for `.DS_Store` 
- I think ignore once is enough.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,9 +21,6 @@ local.properties
 .classpath
 .project
 
-# Windows thumbnail db
-.DS_Store
```
Comment (mhidaka) [inline .gitignore:25]: Good catch! and I prefer It looks better to replace line with `Thumbs.db` instead of just deleting it. 
Could you please do it?

Comment (fumiya-kume) [reply .gitignore:25]: Gotcha, I'm not good at Windows... so much helpful for me... 🙇

Comment (fumiya-kume) [reply .gitignore:25]: Fixed in 30589d54fb12e38d85345cfe5bc398182da7c7d7

**Review by takahirom — APPROVED** [review]
👍

**Pull request merged**